tests/regression-sprint-1.coffee

```diff
@@ -246,7 +246,7 @@ describe '| regression-sprint-1 |', ->
           page.chrome.url (url)->
             url.assert_Contains '/guest/default.html'
             page.open '/user/main.html',(html,$)->
-              $('#loginwall .alert #message').html().assert_Is 'You need to login to see that page.'
+              $('#message').html().assert_Is '<span>You need to login to see that page.</span>'
```

Comment:
Hey @DinisCruz , I thought about using text() instead, so that way we can compare the row text, the problem is if there is a fix to bold that text and we need to make sure that the text is bold :) 

What do you think?